src/test/java/com/kainos/discoverydiary/resources/DiaryResourceTest.java

```diff
@@ -38,8 +38,11 @@ public void listDiaryEntries_whenDiaryEntryAdded_returnsViewWithEntryContained()
         String startDate = "18/06/2016";
         String startTime = "16:00:00";
         String sessionGoal = "Test 1";
+        String imageUrl = "";
+        
+        DiaryEntry diaryEntry = new DiaryEntry(sessionType, title, startDate, startTime, sessionGoal,
+                projectId, imageUrl);
```

Comment:
Could you update this unit test to check that it is setting the image url correctly to the default one and the provided one.